### PR TITLE
User can use a cluster role to access metrics

### PR DESCRIPTION
**Pull request by legionus**
Documentation for https://github.com/openshift/image-registry/pull/73

### CODE REVIEW

**Comment by legionus** [comment]
@bparees please review

**Comment by legionus** [comment]
[(5) Expose registry metrics with openshift auth [registry]](https://trello.com/c/iOKMS2PA/1490-5-expose-registry-metrics-with-openshift-auth-registry)

File: install_config/registry/accessing_registry.adoc [hunk 1]
```diff
@@ -305,4 +306,42 @@ openshift_registry_request_duration_seconds_count{name="test/origin-pod",operati
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
 xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[registry configuration].
 
+Another way how to access metrics is to use a cluster role. You still need to enable endpoint,
```
Comment (bparees) [inline install_config/registry/accessing_registry.adoc]: s/how//
s/access metrics/access the metrics/
s/enable endpoint/enable the endpoint/

**Review by bparees — CHANGES_REQUESTED** [review]
couple nits and lgtm

**Comment by bparees** [comment]
@openshift/team-documentation ptal

File: install_config/registry/accessing_registry.adoc (hunk 1 above)
Comment (legionus) [reply install_config/registry/accessing_registry.adoc]: @bparees done

File: install_config/registry/accessing_registry.adoc [hunk 2]
```diff
@@ -305,4 +306,42 @@ openshift_registry_request_duration_seconds_count{name="test/origin-pod",operati
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
 xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[registry configuration].
 
+Another how to access the metrics is to use a cluster role. You still need to enable the endpoint,
```
Comment (tmorriso-rh) [inline install_config/registry/accessing_registry.adoc]: Another method, process, example, etc.?

File: install_config/registry/accessing_registry.adoc [hunk 3]
```diff
@@ -305,4 +306,42 @@ openshift_registry_request_duration_seconds_count{name="test/origin-pod",operati
...
+but you do not need to specify a `<secret>`. The part of configuration file responsible
+for the metrics should look like this:
```
Comment (tmorriso-rh) [inline install_config/registry/accessing_registry.adoc]: "The part of the configuration file responsible for metrics should look like this:"

**Comment by tmorriso-rh** [comment]
@bparees Few questions/comments. Thanks.

File: install_config/registry/accessing_registry.adoc (hunk 2 above)
Comment (bparees) [reply install_config/registry/accessing_registry.adoc]: heh..  @legionus you removed the wrong work :)

should be "another way to access the metrics...."

Comment (legionus) [reply install_config/registry/accessing_registry.adoc]: @tmorriso-rh `Another method to access the metrics ...` is it ok for you ?

**Comment by legionus** [comment]
@bparees @tmorriso-rh now looks better ?

**Comment by bparees** [comment]
lgtm

**Comment by bmcelvee** [comment]
Thanks! LGTM :sun_with_face: 

When QE has a chance to review, I'll merge

**Pull request merged**